hw/top_earlgrey/syn/asic.constraints.sdc

```diff
@@ -63,7 +63,6 @@ set IO_CLK_PIN u_ast/u_io_clk/u_io_osc/u_buf/out_o
 # target is 96MHz, overconstrain by factor
 set IO_TCK_TARGET_PERIOD 10.416
```

Comment:
I happen to like making this [expr {1000 / 96}] or similar, to remove some "magic numbers" from the scripts.  But a bit orthogonal to what you are doing.